SPECS/cal10n/cal10n.spec

```diff
@@ -16,16 +15,15 @@
 #
 Summary:        Compiler assisted localization library (CAL10N)
 Name:           cal10n
-Version:        0.7.7
-Release:        6%{?dist}
+Version:        0.8.1.10
+Release:        1%{?dist}
 License:        MIT
 Vendor:         Microsoft Corporation
 Distribution:   Azure Linux
 Group:          Development/Libraries/Java
 URL:            http://cal10n.qos.ch
-Source0:        https://github.com/qos-ch/cal10n/archive/refs/tags/v_%{version}.tar.gz#/%{name}-%{version}.tar.gz
-Source1:        build.xml-0.7.7.tar.xz
-Patch0:         cal10n-0.7.7-sourcetarget.patch
+Source0:        https://code.opensuse.org/adrianSuSE/%{name}/blob/factory/f/%{name}-%{version}.tar.xz
+Source1:        %{name}-build.tar.xz
```

Comment:
Please also clarify where this tarball is coming from and how to recreated it later, if needer or during an upgrade.

Reply:
Good point. Added comment and changelog entry to point to the source.

> Please also clarify where this tarball is coming from and how to recreated it later, if needer or during an upgrade.